back up tic

```diff
@@ -0,0 +1,562 @@
+	} else if (board[0][2] === 0){
+		idThree.classList.add("yellow");
+		idThree.innerHTML = "O";
+		round++;
+		buildArray(idThree);
+	}else if (board[2][2] === 0){
+		idNine.classList.add("yellow");
+		idNine.innerHTML = "O";
+		round++;
+		buildArray(idNine);
+	}else if (board[2][0] === 0){
+		idSeven.classList.add("yellow");
+		idSeven.innerHTML = "O";
+		round++;
+		buildArray(idSeven);
+	} else if (board[1][0] === 0){
+		idFour.classList.add("yellow");
+		idFour.innerHTML = "O";
+		round++;
+		buildArray(idFour);
+	} else if (board[0][0] === 0){
+		idOne.classList.add("yellow");
+		idOne.innerHTML = "O";
+		round++;
+		buildArray(idOne);
+	}else if (board[1][2] === 0){
+		idSix.classList.add("yellow");
+		idSix.innerHTML = "O";
+		round++;
+		buildArray(idSix);
+	}else if (board[2][1] === 0){
+		idEight.classList.add("yellow");
+		idEight.innerHTML = "O";
+		round++;
+		buildArray(idEight);
+	}else if (board[0][1] === 0){
+		idTwo.classList.add("yellow");
+		idTwo.innerHTML = "O";
+		round++;
+		buildArray(idTwo);
+	}
+
+};
+
+var clickedBox = function (element){
+	var turn = "";
+	if ((element.classList.contains('blue') === false)&& 
+		(element.classList.contains('yellow') === false)){
+		if (round % 2 === 0){
+			turn = "X";
+			round++;
+			element.classList.add("blue");
+			element.innerHTML = turn;
+			buildArray(element);
+			winner2();
+			if (computerplay === true){
+				computerturn();	
+			}
+		}else {
+			turn = "O";
+			round++;
+			element.classList.add("yellow");
+			element.innerHTML = turn;
+			buildArray(element);
+			winner2();
+			if (computerplay === true){
+				computerturn();	
+			} 
+		}
+	}
+	if ((round === 9) && ((document.getElementById("winner").innerHTML === "O is the Winner!")||
+		(document.getElementById("winner").innerHTML = "X is the Winner!"))) {
+		document.getElementById("winner").innerHTML = "TIE GAME!"
+	}
+	console.log("board is " + board);
+};
+
+var idOne = document.getElementById("1");
+var idTwo = document.getElementById("2");
+var idThree = document.getElementById("3");
+var idFour = document.getElementById("4");
+var idFive = document.getElementById("5");
+var idSix = document.getElementById("6");
+var idSeven = document.getElementById("7");
+var idEight = document.getElementById("8");
+var idNine = document.getElementById("9");
+var idTen = document.getElementById("winner");
+
+idOne.addEventListener("click", function (){clickedBox(idOne);});
+idTwo.addEventListener("click", function (){clickedBox(idTwo);})
+idThree.addEventListener("click", function (){clickedBox(idThree);});
+idFour.addEventListener("click", function (){clickedBox(idFour);});
+idFive.addEventListener("click", function (){clickedBox(idFive);});
+idSix.addEventListener("click", function (){clickedBox(idSix);});
+idSeven.addEventListener("click", function (){clickedBox(idSeven);});
+idEight.addEventListener("click", function (){clickedBox(idEight);});
+idNine.addEventListener("click", function (){clickedBox(idNine);});
```

Comment:
Another suggestion to help eliminate repetition, getting all the boxes using `getElementsByClassName` helps because the function returns an array. Then, the event listeners could be added by looping through the array.